rllib/algorithms/algorithm.py

```diff
@@ -2562,6 +2565,15 @@ def _compile_iteration_results(
     def __repr__(self):
         return type(self).__name__
 
+    def _record_usage(self, config):
+        """Record the framework and algorithm used.
+
+        Args:
+            config: Algorithm config dict.
+        """
+        record_extra_usage_tag(TagKey._RLLIB_FRAMEWORK, config["framework"])
+        record_extra_usage_tag(TagKey._RLLIB_ALGORITHM, self.__class__.__name__)
```

Comment:
Is it possible that multiple frameworks or algorithms are used in one cluster?

Reply:
there should only be 1 algorithm in training.
thanks for the good question.